pkg/backup/restore/restore_manager.go

```diff
@@ -366,6 +366,18 @@ func (rm *restoreManager) makeRestoreJob(restore *v1alpha1.Restore) (*batchv1.Jo
 		},
 	})
 
+	// mount volumes if specified
+	if restore.Spec.Local != nil {
+		localVolume := restore.Spec.Local.Volume
+		localVolumeMount := restore.Spec.Local.VolumeMount
+		if localVolumeMount.Name != localVolume.Name {
```

Comment:
ditto.

Reply:
same reason, no need to validate

Reply:
moved validation logic to `validateBackup()`